src/taskProvider.ts

```diff
@@ -276,18 +276,21 @@ export class TaskProvider implements vscode.TaskProvider {
   private runDevToolsImageExec(): string {
     let exec = "";
 
+    const conf = vscode.workspace.getConfiguration("ledgerDevTools");
```

Comment:
You can put the definition of the `dockerRunArgs` just below like 232 so that it could be used by other tasks as well, don't you think ?

Reply:
Done in b47a1a23540891afee4c91ba90cab3b2dd2823ae